doc/updating_documentation/How_to_update_this_documentation.md

```diff
@@ -0,0 +1,26 @@
+# How to update this documentation
+
+## Documentation update procedure
+
+1. Make a pull request to master branch in repository Juniper/contrail-windows-docs with required changes.
+2. Once PR is accepted, an Administrator should deploy new modifications to gh-pages branch.
+
+## How to preview modifications
+
+1. Install MkDocs (see below).
+2. Apply required modifications to the documentation.
+3. Invoke `mkdocs serve` in main repository directory. It's going to start local HTTP server with documentation preview. Follow instructions displayed by mkdocs to see the outcome.
+
+## How to install MkDocs
+
+MkDocs requires Python (both 2 and 3) and PIP. Follow instructions on [http://www.mkdocs.org/#installation](http://www.mkdocs.org/#installation). Basically `pip install mkdocs` should work.
+
+## How to deploy modifications
+
+github.io fetches HTML files generated by mkdocs from gh-pages branch of repository Juniper/contrail-windows-docs. Administrator priviledges are required to modify gh-pages branch.
+
+To deploy new modifications, the following procedure should be applied:
+
+1. Install MkDocs.
+2. Checkout master branch of repository Juniper/contrail-windows-docs.
+3. Invoke `mkdocs gh-deploy` in main repository directory and follow displayed instructions. Additional details are described in [http://www.mkdocs.org/user-guide/deploying-your-docs/](http://www.mkdocs.org/user-guide/deploying-your-docs/).
```

Comment:
'Additional details' should be moved to a `Note:` section below this list to make it shorter. Also, it's not actionable, so it shouldn't be on list of actions.

Reply:
This remark refers specifically to this action.